assignments/week_2/exercise_2_1/exercise_2_1.cpp

```diff
@@ -0,0 +1,5 @@
+#include <vector>
+
+int main() {
+
```

Comment:
Maybe leave the `readInVector` stuff? Or do we just assume they'll copy their entire solution in here? If it's the latter I'd leave a comment to mention that.

Reply:
Leaving this open to make sure the change is sufficient.